src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -48,6 +48,10 @@
   <module name="org.openhab.tools.analysis.checkstyle.OverridingParentPomConfigurationCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
+      <property name="severity" value="warning" />
```

Comment:
Is it possible to deactivate certain checks in a bundle? If so, I would prefer it to be an error and if there is the situation where it really has to be there, it could be deactivated in the pom.xml of the according bundle.